prog.go

```diff
@@ -417,6 +417,10 @@ var haveProgTestRun = internal.FeatureTest("BPF_PROG_TEST_RUN", "4.12", func() e
 		// otherwise misdetect due to insufficient permissions.
 		return internal.ErrNotSupported
 	}
+	if errors.Is(err, unix.EINTR) {
+		// Issue #203: we know that PROG_TEST_RUN is supported if we get EINTR.
```

Comment:
Nit: are we referencing other GH issues in the code comments? I would prefer that the comment is just "We know that PROG_TEST_RUN is supported if we get EINTR". Unless the precedent has been set already, then I'm fine.

Reply:
Fixed